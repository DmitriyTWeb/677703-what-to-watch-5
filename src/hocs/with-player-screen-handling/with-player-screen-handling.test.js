import React from "react";
import renderer from "react-test-renderer";
import withPlayerScreenHandling from "./with-player-screen-handling";

const MockComponent = () => {
  return (
    <div>
    </div>
  );
};

const MockComponentWrapped = withPlayerScreenHandling(MockComponent);

it(`withPlayerScreenHandling is rendered correctly`, () => {
  const tree = renderer.create(
      <MockComponentWrapped
        history={{}}
      />
  );
  expect(tree).toMatchSnapshot();
});

