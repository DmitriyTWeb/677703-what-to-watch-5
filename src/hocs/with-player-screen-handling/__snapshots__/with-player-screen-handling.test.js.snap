// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withPlayerScreenHandling is rendered correctly 1`] = `<div />`;
